documentation/src/developer_documentation/archie_and_fuzzy.md

```diff
@@ -0,0 +1,54 @@
+# What and How?
+
+Fuzzy and Archie are two servers hosted in the ChainSafe office. Both belong to
+the Forest team.
+
+Archie and Fuzzy are accessible through a CloudFlare tunnel. Add this to your
+SSH config (`~/.ssh/config`):
+
+```
+Host archie.chainsafe.dev
+  ProxyCommand /usr/local/bin/cloudflared access ssh --hostname %h
+  User archie
+
+Host fuzzy-forest.chainsafe.dev
+  ProxyCommand /usr/local/bin/cloudflared access ssh --hostname %h
+  User fuzzy
+```
+
+You should be able to directly ssh into `archie.chainsafe.dev` and
```

Comment:
It may be obvious, but let's mention that the SSH key needs to be manually added to the `authorized_keys` for new users

Reply:
Updated.